packages/ingest/test/test-http.js

```diff
@@ -35,9 +39,10 @@ class MyTestDiscoveryClass {
 class MyTestHttpDiscoveryClass extends TestHttpMixin(MyTestDiscoveryClass) {}
 const myTestHttpDiscoveryClass = new MyTestHttpDiscoveryClass();
 
-test('Download remote file to s3', async (t) => {
+test('Download remote file to s3 with correct content-type', async (t) => {
   const bucket = randomString();
   const key = randomString();
+  const expectedContentType = 'application/x-hdf';
```

Comment:
Why is the expected content type here different than in the FTP unit tests above?

Reply:
This works with `key = randomString();` because the http `sync()` code checks the http provider instead of inferring the `ContentType`.